### PR TITLE
Make the ComponentPicker independent of the Toolbar

### PR DESCRIPTION
**Pull request by EgonBolton**
- I moved the Toolbar dialogs to their respective Plugins (TablePlugin, ImagesPlugin, etc.).
- converted Toolbar.css into a new UI component: dialog

The main reason for making this change is that the ComponentPicker plugin relied on Toolbar to import the dialogs, when in fact they should be separate plugins. It is now possible for someone to use the ComponentPicker plugin without using the Toolbar plugin.

### CODE REVIEW

**Review by tylerjbainbridge — APPROVED** [review]
Great idea! Thanks for this

File: packages/lexical-playground/src/plugins/TablePlugin.tsx [hunk 1]
```diff
@@ -88,6 +91,58 @@ export function TableContext({children}: {children: JSX.Element}) {
   );
 }
 
+export function InsertTableDialog({
```
[the remark this answers is not in the document]
Comment (EgonBolton) [reply packages/lexical-playground/src/plugins/TablePlugin.tsx:94]: Yes. I noticed that. I saw that in the Playground in the insert options "Table" and "Table (experimental)" appear. So I assumed that someone was working on it and that's why I left it as it was.

[the remark this answers is not in the document]
Comment (acywatson) [reply packages/lexical-playground/src/plugins/TablePlugin.tsx:94]: Ok, that's cool. Looks like we have a few failing tests, but happy to merge once those are fixed. Thank you!

[the remark this answers is not in the document]
Comment (EgonBolton) [reply packages/lexical-playground/src/plugins/TablePlugin.tsx:94]: I'm still struggling to debug a specific test locally. Either way, I think I already fixed the bug in the last commit. Could you check it?

[the remark this answers is not in the document]
Comment (acywatson) [reply packages/lexical-playground/src/plugins/TablePlugin.tsx:94]: @EgonBolton yep! Running tests - after we merge this you won't be a first-time contributor anymore and they should auto-run. Sorry for the inconvenience.

[the remark this answers is not in the document]
Comment (EgonBolton) [reply packages/lexical-playground/src/plugins/TablePlugin.tsx:95]: Perfect @acywatson. Can you run the tests again?

**Pull request merged**